requirements.yaml

```diff
@@ -0,0 +1,6 @@
+dependencies:
+
+  - name: postgresql
+    version: "9.4.*"
```

Comment:
wondering how much testing you've done with this? I know we run against 9.6 currently, wasn't sure how things would work with 9.4

Reply:
This is the version of the chart - it's actually `v11.9.*`. And yeah - it's OK.

Reply:
I've tested most use cases - incl. also running multiple PostgreSQL Read Replicas.